LondonBritishLibrary/orient/BLorient11763.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BLorient11763">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Magical Scroll</title>
```

Comment:
I have a vague feeling that it is better to say "magic scroll", not "magical", as it contains magic, and not works magic (for the same reason for me "magical prayer" sounds better than "magic prayer"). I have seen both used though. @AaronButts as the only native speaker around may have a better idea. Does it make a difference at all? Should we aspire to any sort of standard?

Reply:
For me, "magic scroll" is much much better (I don't use "magical scroll"). As Genia says, it is a scroll that contains magic. 

I would also personally use "magic prayer", following similar logic: It is a prayer consisting of magic. 

I realize that "magical" is etymologically an adjective for "magic", but it has quite different connotations in usage: supernatural, as well as remarkable, exceptional, incredible. Basically, for me "magical" has some value judgement, whether about the efficacy of the magic, or more generally that something is extraordinary. I personally try to avoid using "magical" as much as possible when describing the practice of magic.